GDB-6528: Scheduled GraphDB backups (#111)
Ontotext-AD/graphdb-helm: examples/backup-local/README.md

# Local GraphDB Backups Example

> This folder contains a simple example of how to configure the Helm chart with local GraphDB backups backed by an existing Persistence Volume

Reviewer: ```suggestion
This example shows how to configure the Helm chart with local GraphDB backups backed by an existing Persistence Volume
```

Claim.

## Configuration

> The default [values.yaml](values.yaml) in the example uses a simple PVC that should be sufficient for demonstration purposes,

Reviewer: ```suggestion
The default [values.yaml](values.yaml) in the example uses a simple PVC that should be sufficient for demonstration purposes.
```

> However, ideally, you should consider using a PVC with cross-zone or cross-region data replication for better resiliency.

Reviewer: Either "however" or "ideally".

> Have in mind that the chart does not rotate backups, this is up to you and your retention policy.

Reviewer: I always find "have in mind" a bit awkward. I think keep / bear in mind are more common.
